hazelcast/src/main/java/com/hazelcast/partition/PartitionLostListener.java

```diff
@@ -22,32 +22,45 @@
 import java.util.EventListener;
 
 /**
- * PartitionLostListener provides the ability to be notified upon a possible data loss when a partition loses a replica.
+ * PartitionLostListener provides the ability to be notified upon a
+ * possible data loss when a partition loses a replica.
  * <p>
- * When the cluster is initialized, each node becomes owner of some of the partitions, and backup of the some other partitions.
- * We call owner of partition "primary replica" and backup nodes "backup replicas". Partition replicas are ordered.
- * A primary replica node keeps all data that is mapped to the partition. If a Hazelcast data structure is configured with
- * 1 backup, its data is put into the primary replica and the first backup replica. Similarly, data of a Hazelcast data structure
- * that is configured with 2 backups is put into the primary replica, the first backup replica, and the second backup replica.
- * The idea is same for higher backup counts.
+ * When the cluster is initialized, each node becomes owner of some of the
+ * partitions, and backup of the some other partitions. We call owner of
+ * partition "primary replica" and backup nodes "backup replicas".
+ * Partition replicas are ordered. A primary replica node keeps all data
+ * that is mapped to the partition. If a Hazelcast data structure is
+ * configured with 1 backup, its data is put into the primary replica and
+ * the first backup replica. Similarly, data of a Hazelcast data structure
+ * that is configured with 2 backups is put into the primary replica, the
+ * first backup replica, and the second backup replica. The idea is same
+ * for higher backup counts.
  * <p>
- * When a node fails, primary replicas of its partitions are lost. In this case, ownership of each partition owned by
- * the unreachable node is transferred to the first available backup node. After this point, other backup nodes sync themselves
- * from the new partition owner node in order to populate the missing backup data.
+ * When a node fails, primary replicas of its partitions are lost. In this
+ * case, ownership of each partition owned by the unreachable node is
+ * transferred to the first available backup node. After this point, other
+ * backup nodes sync themselves from the new partition owner node in order
+ * to populate the missing backup data. This sync is only happen when backup
+ * partition replica versions are not equal to the primary ones.
```

Comment:
only added this last sentence:
`This sync is only happen when backup partition replica versions are not equal to the primary ones`